### DISCUSSION TITLE
How to unittest custom endpoints.

### DISCUSSION THREAD

**Discussion by goffity** [directus/directus#15755] [Archive]
I am a newbie in Javascript and Directus.
When writing customer endpoint on Directus
```
module.export = function f(){
...
router.get(...);
}
```

How to test that function written, or have a logical validation process?

**Comment by rijkvanzanten** [comment]
The router object passed to the function parameter is just a regular Express router object, so you can unit test your route handlers just like you'd unit test any other express route 👍🏻
